### PR TITLE
Added new docker image info in README

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Added new docker image info in README]
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,22 @@
...
+an installation of oneview-sdk-ruby and you can use it by just pulling down the Docker Image:
+The Docker Store image tag consist of two sections: <sdk_version-OV_version>
+
+```bash
+# Download and store a local copy of oneview-sdk-ruby and
+# use it as a Docker image.
+$ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-ruby:v5.13.0-OV5.2
+# Run docker commands below given, which  will in turn create
+# a sh session where you can create files, issue commands and execute the recipes.
+$ docker run -it hewlettpackardenterprise/hpe-oneview-sdk-for-chef:v5.13.0-OV5.2 /bin/sh
```
Comment (chebroluharika) [inline README.md:24]: it should be hpe-oneview-sdk-for-ruby:v5.13.0-OV5.2

File: README.md [hunk 2]
```diff
@@ -9,6 +9,22 @@
...
+We also provide a lightweight and easy way to test and execute `oneview-sdk-ruby`. The `hpe-oneview-sdk-for-ruby:<tag>` docker image contains
+an installation of oneview-sdk-ruby and you can use it by just pulling down the Docker Image:
+The Docker Store image tag consist of two sections: <sdk_version-OV_version>
+
+```bash
+# Download and store a local copy of oneview-sdk-ruby and
+# use it as a Docker image.
+$ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-ruby:v5.13.0-OV5.2
+# Run docker commands below given, which  will in turn create
+# a sh session where you can create files, issue commands and execute the recipes.
```
Comment (chebroluharika) [inline README.md:23]: ruby doesn't have recipes. Please change it to examples.

Comment (nabhajit-ray) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (nabhajit-ray) [reply README.md]: Done

**Pull request merged**